user/api.go

```diff
@@ -357,12 +358,20 @@ func (a *Api) GetUsers(res http.ResponseWriter, req *http.Request) {
 	} else if role := req.URL.Query().Get("role"); role != "" && !IsValidRole(role) {
 		a.sendError(res, http.StatusBadRequest, STATUS_INVALID_ROLE)
 
+	} else if authenticated := req.URL.Query().Get("authenticated"); authenticated != "" && authenticated != "false" && authenticated != "true" {
```

Comment:
You could use the error return of strconv.ParseBool() to see if the value is valid ?

Reply:
Missing update on the OpenAPI documentation

Reply:
done